changelogs/fragments/pr-171-envvar-for-mount-point.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- auth methods - Add support for configuring the ``mount_point`` auth method option in plugins via the ``HASHI_VAULT_MOUNT_POINT`` environment variable, ``ansible_hashi_vault_mount_point`` ansible variable, or ``mount_point`` INI section (https://github.com/ansible-collections/community.hashi_vault/pull/171).
```

I messed up the wording on this 🤦‍♂️ I'm going to merge this and "fix it in post" since I am on a short timeline for the release of 1.5.0 and then 2.0.0.
```suggestion
- auth methods - Add support for configuring the ``mount_point`` auth method option in plugins via the ``ANSIBLE_HASHI_VAULT_MOUNT_POINT`` environment variable, ``ansible_hashi_vault_mount_point`` ansible variable, or ``mount_point`` INI section (https://github.com/ansible-collections/community.hashi_vault/pull/171).
```